static/js/components/TopSources.jsx

```diff
@@ -49,6 +49,9 @@ const TopSources = () => {
                   &nbsp;view(s)
                 </em>
               </span>
+              <span>
+                <img src={public_url} alt="" />
```

Comment:
The Top Sources widget is going to take up a small fraction of the homepage, so large postage stamps doesn't make sense. So the img tag should probably be set a fraction of the widget height/width. You can do this with css styling or inline.

Reply:
Probably also makes sense to have an `alt` text with the source name (which will make this more accessible and resilient).

Reply:
In addition to @profjsb's suggestions, I'd suggest replacing the bullet points that are currently there with the small (~100x100px) images and letting the text float to the right of those.

Reply:
Yes excellent points. Should be improved.